www/vercel.json

```diff
@@ -1,5 +1,40 @@
 {
   "redirects": [
+    {
+      "source": "/docs/server/output-validation",
+      "destination": "/docs/server/validators",
+      "permanent": true
+    },
+    {
+      "source": "/docs/server/procedures#with-scale-ts",
+      "destination": "/docs/server/validators#input-validators",
+      "permanent": true
+    },
+    {
+      "source": "/docs/server/procedures#with-superstruct",
```

Comment:
You can't add hash (`#` ) in `source` AFAIK since it's a server-side redirect

Reply:
Yea that's my bad and makes total sense...

I have redirects with hashes but those have the hash in the destination...

Reply:
Julius I can't believe you've done this to me 😂

I've accepted the suggestion from Alex. Will remember it for next time